akita/src/plugins/stateHistory/stateHistoryPlugin.ts

```diff
@@ -3,12 +3,19 @@ import { toBoolean } from '../../toBoolean';
 import { AkitaPlugin, Queries } from '../plugin';
 import { EntityParam } from '../entityCollectionPlugin';
 import { logAction } from '../../actions';
+import { isFunction } from '../../isFunction';
 
 export interface StateHistoryParams {
   maxAge?: number;
   comparator?: (prevState, currentState) => boolean;
 }
 
+export type History<S> = {
+  past: S[],
+  present: S,
```

Comment:
present should be `S | null`

Reply:
Ok. What should the commit message for an adjustment like that look like?

Reply:
use the amend command. https://medium.com/@igor_marques/git-basics-adding-more-changes-to-your-last-commit-1629344cb9a8